src/js/components/BreadcrumbComponent.jsx

```diff
@@ -1,7 +1,12 @@
+var _ = require("underscore");
+var classNames = require("classnames");
 var React = require("react/addons");
 var Link = require("react-router").Link;
 
 var PathUtil = require("../helpers/PathUtil");
+var StyleDimensions = require("../constants/StyleDimensions");
```

Comment:
Sure that we will use this in the future more then at one place?
Otherwise put this directly into this component to simplify the things.